xbout/plotting/plotfuncs.py

```diff
@@ -169,6 +196,24 @@ def plot2d_wrapper(da, method, *, ax=None, separatrix=True, targets=True,
         extend = kwargs.get('extend', 'neither')
         fig.colorbar(artists[0], ax=ax, extend=extend)
 
+    if gridlines is not None:
+        if gridlines is True:
+            gridlines = (1, 1)
+        if not isinstance(gridlines, collections.Sequence):
```

Comment:
`collections.sequence` seems an odd choice of container class? We just need to store either one or two ints right? Why not just a short list? Or even a dictionary?
```python
gridlines= {'x': 1, 'y':1}
```

Reply:
Sorry, this was written quickly. Restricting to `int`, `slice` or `dict of int or slice` is better.